src/System.Private.CoreLib/src/System/Exception.CoreCLR.cs

```diff
@@ -137,7 +139,7 @@ private MethodBase GetExceptionMethodFromStackTrace()
             return RuntimeType.GetMethodBase(method);
         }
 
-        public MethodBase TargetSite
+        public MethodBase? TargetSite
```

Comment:
Exception.Message is another one we should add to the "should it be nullable" list (it wasn't changed in the PR so I can't comment on it).  @safern, could you set up a meeting to work through it with all the relevant parties?  We're going to hit more of these.  I've never seen null returned from Message, though (whereas I have from ToString).

Reply:
Done! I think it makes sense to at least return an empty string from `Exception.Message` -- I don't think it makes sense to return null, but that takes us back to the same discussion of `ToString`. Let's discuss this on the meeting and hopefully we all agree on something in order to keep making progress.

Reply:
Going to merge as this one seems "clearer" to me that it shouldn't return null. However, we can revisit tomorrow in our meeting and fix if that is the case.